Extension/src/LanguageServer/extension.ts

```diff
@@ -403,19 +403,26 @@ export function registerCommands(enabled: boolean): void {
     commandDisposables.push(vscode.commands.registerCommand('C_Cpp.ExtractToFreeFunction', enabled ? () => onExtractToFunction(true, false) : onDisabledCommand));
     commandDisposables.push(vscode.commands.registerCommand('C_Cpp.ExtractToMemberFunction', enabled ? () => onExtractToFunction(false, true) : onDisabledCommand));
     commandDisposables.push(vscode.commands.registerCommand('C_Cpp.ExpandSelection', enabled ? (r: Range) => onExpandSelection(r) : onDisabledCommand));
-    commandDisposables.push(vscode.commands.registerCommand('C_Cpp.getIncludes', enabled ? (maxDepth: number) => getIncludes(maxDepth) : onDisabledCommand));
+    commandDisposables.push(vscode.commands.registerCommand('C_Cpp.getIncludes', enabled ? (maxDepth: number) => getIncludes(maxDepth) : onLowImpactDisabledCommand));
 }
 
-function onDisabledCommand() {
-    const message: string = localize(
+function getDisabledCommandMessage(): string {
+    return localize(
         {
             key: "on.disabled.command",
             comment: [
                 "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered."
             ]
         },
         "IntelliSense-related commands cannot be executed when `C_Cpp.intelliSenseEngine` is set to `disabled`.");
-    return vscode.window.showWarningMessage(message);
+}
+
+function onDisabledCommand() {
+    return vscode.window.showWarningMessage(getDisabledCommandMessage());
+}
+
+async function onLowImpactDisabledCommand() {
+    console.log(getDisabledCommandMessage());
```

Comment:
Do you even want to log at all? My vote would be to leave `onDisabledCommand` alone and just create your own 'no-op' handler for the disabled case.

Reply:
(because I'm expecting that the console will be flooded with these messages by copilot and they won't be useful to anyone)

Reply:
Sounds good, I can simplify it.

Reply:
Yeah, I agree with Bob.

Reply:
Changed to no-op handler.